internal/attestation/error.go

```diff
@@ -0,0 +1,57 @@
+package attestation
+
+import (
+	"errors"
+	"fmt"
+)
+
+var (
+	ErrAttester                = errors.New("attester")
+	ErrAttesterUserDataTooLong = fmt.Errorf("%w: user data too long", ErrAttester)
+	ErrVerifier                = errors.New("verifier")
+	ErrVerifierDebugMode       = fmt.Errorf("%w: debug mode", ErrVerifier)
+	ErrVerifierMeasurement     = fmt.Errorf("%w: measurement", ErrVerifier)
+	ErrVerifierNonce           = fmt.Errorf("%w: nonce", ErrVerifier)
+	ErrVerifierTimestamp       = fmt.Errorf("%w: timestamp", ErrVerifier)
+)
+
+func wrapError(baseErr error, msg string, err error) error {
```

Comment:
Defined the error chain format to be "base: message (optional): inner error (optional)". For example
`
Error "verifier: measurement: pcr '1' mismatch: expected '123456789' got '987654321"
`